dist/OBS-statistics.R

```diff
@@ -0,0 +1,291 @@
+# We use ggplot2, a library that makes painting graphics much easier.
+# You can check the types of graphs and examples code here: https://plot.ly/ggplot2
+
+# REQUIRE NEEDED LIBRARIES, in case that you don't have then installed, use: install.packages("ggplot2")
```

Comment:
I think that if we want to allow a user to view statistical graphs then we should create a trello card to create a web interface to view these graphs.